### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by albertnbrown**
new Goerli deploy data and some updated scripts

### CODE REVIEW

**Comment by StoyanD** [comment]
can u remove the old deploys from the .openzepplin folder, and just tag the release as v0.2 when it merges to master. we can get the old deploy plugin gen files from tag v0.1 when we need to.

linter also failing, but everything else looks fine

**Comment by albertnbrown** [comment]
ok, sounds good

File: scripts/sendToL1.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    hre.ethers.utils.arrayify('0x1234')
+  )
+  await tx.wait()
+  return tx
+}
+
+async function setupOP(): Promise<CrossChainMessenger> {
+  hre.changeNetwork(L2_NETWORK)
+  const l2ChainId = hre.network.config.chainId
+  const [l2Wallet] = await hre.ethers.getSigners()
```
Comment (StoyanD) [inline scripts/sendToL1.ts:57]: l2Wallet and l1Wallet are the same signer right?

File: scripts/transferOwnershipL2Bridge.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  const proxyAdmin = await getProxyAdmin()
+
+  const currentOwner = await proxyAdmin.owner()
+  const [me] = await hre.ethers.getSigners()
+  if ((await me.getAddress()) !== currentOwner) {
+    throw new Error('you need to own the proxy admin to run this script')
+  }
+
+  await proxyAdmin.transferOwnership(l2BridgeProxyAddress)
+  console.log(`admin owner changed to ${l2BridgeProxyAddress}`)
```
[the remark this answers is not in the document]
Comment (albertnbrown) [reply scripts/transferOwnershipL2Bridge.ts:19]: fair enough

File: scripts/upgradeL2Eco.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  const L2ECOTokenFactory = await hre.ethers.getContractFactory('L2ECO')
+  const newL2ECOTokenImpl = await L2ECOTokenFactory.deploy()
+  await newL2ECOTokenImpl.deployed()
+  console.log(`new L2 Bridge deployed to ${newL2ECOTokenImpl.address}`)
+
+  hre.changeNetwork(L1_NETWORK)
+
+  const tx = await bridge.upgradeECO(newL2ECOTokenImpl.address, l2gas)
+  await tx.wait()
```
[the remark this answers is not in the document]
Comment (albertnbrown) [reply scripts/upgradeL2Eco.ts:49]: yeah, the cross-chain message takes too long to propagate that it was not useful to have the script sleep for long enough. Instead just use the `getImpl.ts` script to check the change.

[the remark this answers is not in the document]
Comment (albertnbrown) [reply scripts/upgradeL2Eco.ts:49]: to give you an idea, I put in a one minute sleep and that still wasn't consistently enough

File: scripts/upgradeL2Bridge.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  const L2ECOBridgeFactory = await hre.ethers.getContractFactory('L2ECOBridge')
+  const newL2ECOBridgeImpl = await L2ECOBridgeFactory.deploy()
+  await newL2ECOBridgeImpl.deployed()
+  console.log(`new L2 Bridge deployed to ${newL2ECOBridgeImpl.address}`)
+
+  hre.changeNetwork(L1_NETWORK)
+
+  const tx = await bridge.upgradeL2Bridge(newL2ECOBridgeImpl.address, l2gas)
+  await tx.wait()
+  console.log(`L2 Bridge upgraded`)
```
Comment (nknavkal) [inline scripts/upgradeL2Bridge.ts:51]: assuming we don't check if the impl is changed bc the messenger takes too long?

**Review by nknavkal — APPROVED** [review]

File: scripts/upgradeL2Bridge.ts (hunk 4 above)
Comment (albertnbrown) [reply scripts/upgradeL2Bridge.ts:51]: yeah, check my convo with Stoyan above

Comment (StoyanD) [reply scripts/upgradeL2Bridge.ts:51]: can u comment somewhere in there y?

File: scripts/upgradeL2Eco.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (StoyanD) [reply scripts/upgradeL2Eco.ts:49]: can u comment somewhere in there y?

**Pull request merged**